proofpoint_url_defense/plugin.spec.yaml

```diff
@@ -37,3 +37,8 @@ actions:
         description: Decoded Proofpoint URL
         type: string
         required: false
+      decode_success:
```

Comment:
Minor nit but can we change this to `decoded`, update docs, regen? We use that in a few other plugins